code/game/objects/structures/morgue.dm

```diff
@@ -266,17 +266,13 @@ GLOBAL_LIST_EMPTY(crematoriums)
 		update_icon()
 
 		for(var/mob/living/M in conts)
-			if (M.stat != DEAD)
+			if (M.stat == DEAD	|| M.name == "Punishing bird")	//Only kill dead people and punishing bird
+				M.death(1)
```

Comment:
What's the point of it if mob is already dead?

Reply:
Good point, sry i coded it at like 6am